src/test/java/teammates/test/cases/action/InstructorFeedbackResultsDownloadActionTest.java

```diff
@@ -106,6 +116,20 @@ public void testExecuteAndPostProcess() throws Exception {
         assertEquals(expectedFileName, result.getFileName());
         verifyFileContentForSession1InCourse1WithinSection1(result.getFileContent(), session);
 
+        ______TS("Failure case to mock ExceedingRangeException: data is too large to be downloaded in one go");
+
+        action = getAction(paramsWithLargeData);
+        RedirectResult r = getRedirectResult(action);
+
+        expectedDestination = getPageResultDestination("/page/instructorFeedbackResultsPage", true, "");
```

Comment:
This exact value is stored in `Const.ActionUri.INSTRUCTOR_FEEDBACK_RESULTS_PAGE`